init.sh.tpl

```diff
@@ -4,5 +4,5 @@ echo "SRE_TOOLING_SERVER_GROUP=\"${group}\"" >> /etc/default/sre-tooling
 echo "SRE_TOOLING_REGION=\"${region}\"" >> /etc/default/sre-tooling
 
 if [ "${run_mybatis_migrations}" == "true" ] ; then
-  /opt/mybatis/mybatis-migrations-${mybatis_version}/bin/migrate up --path=/tmp/opensrp/configs/assets/migrations --env=deployment
+  /opt/mybatis/mybatis-migrations-${mybatis_version}/bin/migrate up --path=/opt/opensrp/configs/assets/migrations --env=deployment
```

Comment:
Would it be okay to remove the `/opt/opensrp` directory at this point? `rm -r /opt/opensrp`

Reply:
Done. Added a -f since there are some files in the .git directory that require that (or a confirmation from the user).